packages/ui-focusable/src/Focusable/index.tsx

```diff
@@ -73,12 +83,12 @@ class Focusable extends Component<FocusableProps> {
 
   state = {
     focused: false,
-    focusable: false
-  }
+    focusable: undefined
```

Comment:
This was really messy. It could either have a boolean value (I think only false?) or the `HTMLElement` that is focusable. I've rewritten the code so its value can be `HtmlElement` or `undefined`

Reply:
Just a question: what is the practical difference between this (`as FocusableState`) and this?
```
state: FocusableState = {
  focused: false,
  focusable: undefined
}
```
As far as I could tell so far the second is telling the variable what type to allow, the `as` cast is a more forceful action, and can even hide possible tiny bugs, that's why I'm trying to avoid using it whenever I can. 

Can we talk about this on the next standup? It probably isn't a breaking issue here, just wanna discuss it in general (Interesting article: https://dev.to/ylerjen/typescript-cast-is-a-type-breaker-1jbh)

Reply:
this is here by accident, I've added it as normal type not as a cast